pkg/apiserver/archive/experiment.go

```diff
@@ -81,3 +83,39 @@ func (s *Service) listExperiments(c *gin.Context) {
 
 	c.JSON(http.StatusOK, data)
 }
+
+// @Summary Get the details of chaos experiment.
+// @Description Get the details of chaos experiment.
+// @Tags archives
+// @Produce json
+// @Param namespace query string false "namespace"
+// @Param name query string false "name"
+// @Param kind query string false "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Success 200 {array} core.ArchiveExperiment
+// @Router /api/archives/report [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) experimentDetail(c *gin.Context) {
+	kind := c.Query("kind")
+	name := c.Query("name")
+	ns := c.Query("namespace")
+
+	if kind == "" || name == "" || ns == "" {
+		c.Status(http.StatusBadRequest)
+		_ = c.Error(utils.ErrInvalidRequest.New("kind, name or namespace is empty"))
+		return
+	}
```

Comment:
That's the problem we have talked about at the start of design. Since archives are collections of experiments in the database, we can only use id to uniquely identify one experiment. Using `kind`, `name` and `namespace` is not sufficient.

TBH I don't know how can we get the ID.

Reply:
Later we plan to add suffix to name when creating chaos to ensure that name is unique.